api/tests/acceptance/scripts/add-many-tudents-to-sco-certification-center_test.js

```diff
@@ -0,0 +1,46 @@
+const { expect } = require('../../test-helper');
+const { MIDDLE_SCHOOL_ID } = require('../../../db/seeds/data/organizations-sco-builder');
+
+const { knex } = require('../../../lib/infrastructure/bookshelf')
+const BookshelfSchoolingRegistration = require('../../../lib/infrastructure/data/schooling-registration');
+const { databaseBuilder } = require('../../test-helper');
+
+const { addManyStudentsToScoCertificationCenter } = require('../../../scripts/add-many-students-to-sco-certification-center');
+
+describe('Acceptance | Scripts | add-many-students-to-sco-certification-centers.js', () => {
+
+  describe('#addManyStudentsToScoCertificationCenter', () => {
+
+    const getNumberOfSchoolingRegistrations = () => {
+      return BookshelfSchoolingRegistration.count()
+        .then((number) => parseInt(number, 10));
+    };
```

Comment:
Proposition : sortir cette fonction des describe et préfixer par un `_` :
```suggestion
    const _getNumberOfSchoolingRegistrations = () => {
      return BookshelfSchoolingRegistration.count()
        .then((number) => parseInt(number, 10));
    };
```